server/app/controllers/applicant/ApplicantInformationController.java

```diff
@@ -67,7 +68,8 @@ public ApplicantInformationController(
   @Secure
   public CompletionStage<Result> setLangFromBrowser(Http.Request request, long applicantId) {
 
-    CompletionStage<Optional<String>> applicantStage = this.applicantService.getName(applicantId);
+    CompletionStage<ApplicantPersonalInfo> applicantStage =
+        this.applicantService.getPersonalInfo(applicantId);
```

Comment:
Is the value of this CompletionStage ever used?

Reply:
Nope, removing